docs/faq.rst

```diff
@@ -685,6 +687,33 @@ The above will block flagged IPs for a week, you can of course change it to you
 
 Using DOCKER-USER chain ensures that the blocked IPs are processed in the correct order with Docker. See more in: https://docs.docker.com/network/iptables/
 
+Option 2:  Use ipset together with iptables
+IMPORTANT: You have to install ipset on the host system, eg. `apt-get install ipset` on a Debian/Ubuntu system.
+
+See ipset homepage for details on ipset, https://ipset.netfilter.org/.
+
+ipset and iptables provide one big advantage over just using iptables: This setup reduces the overall iptable rules.
+There is just one rule for the bad authentications and the IPs are within the ipset. 
+Specially in larger setups with a high amount of brute force attacks this comes in handy.
+Using iptables with ipset might reduce the system load in such attacks significantly.
+
+.. code-block:: bash
+
+  [Definition]
+
+  actionstart = actionstart = ipset --create f2b-bad-auth iphash
+                iptables -I DOCKER-USER -p tcp -m multiport --dports 1:1024 -m set --match-set f2b-bad-auth src -j DROP
+
+  actionstop = iptables -D DOCKER-USER -p tcp -m multiport --dports 1:1024 -m set --match-set f2b-bad-auth src -j DROP
```

Comment:
Why do you restrict it to TCP and lower port numbers?

I don't understand why port zero is excluded either :)

I would keep it simple and make it something like: ``iptables -I DOCKER-USER -m set --match-set f2b-bad-auth src -j DROP``

Reply:
I did match it to the existing documentation:
> actionstart = iptables -N f2b-bad-auth
>              iptables -A f2b-bad-auth -j RETURN
>              iptables -I DOCKER-USER -p tcp -m multiport --dports 1:1024 -j f2b-bad-auth

I did it different for my own setup, with a more configurable approach - but this would be different to anything that was documented before. It's more complex (with additional bash scripts called) and might be a bit confusing if you'll never dealt with fail2ban. 

It's documented here https://tkaefer.de/blog/2021/12/07/follow-up-docker-and-fail2ban-how-i-solved-it-for-me/

As said, it's more complex and might be too much for an FAQ.

Reply:
That's a good reason :)

IMHO it's very confusing and error prone to selectively block a subset of ports... Please update the existing documentation too and let's make the block "global" (all protocols, all ports)